packages/cli/src/commands/sandbox/sandbox_command.test.ts

```diff
@@ -105,7 +105,7 @@ void describe('sandbox command', () => {
     assert.match(output, /--dir-to-watch/);
     assert.match(output, /--exclude/);
     assert.match(output, /--format/);
```

Comment:
Renamed to `config-format` in [ece9635](https://github.com/aws-amplify/amplify-backend/pull/643/commits/ece963513ce537adfccc22bd0e4b171674f31ece)

Reply:
I'm okay with e2e tests going in a follow up PR. When you do get to that, I suggest adding this as an additional assertion on the existing sandbox test rather than a whole new test.